phpBB/viewtopic.php

```diff
@@ -48,6 +48,14 @@
 
 $update		= $request->variable('update', false);
 
+//extract forum image
+	$sql = 'SELECT forum_image
```

Comment:
FYI for the pr author: forum images are really just forum icons set by the admin. Not sure that using the forum image is what one would want to do here either instead of just using say favicon or the board's logo.